coding_style/README.adoc

```diff
@@ -153,6 +153,22 @@ Rationale:: https://yaml.org/type/bool.html[YAML 1.1] allows all variants wherea
   But do use quotes for user-side strings such as descriptions, names, and messages.
 * Even if JSON is valid YAML and Ansible understands it, do only use JSON syntax if it makes sense (e.g. a variable file automatically generated) or adds to the readability.
   In doubt, nobody expects JSON so stick to YAML.
+* Break up lengthy Jinja templates into multiple templates when there are distinct logical sections.
++
+[%collapsible]
+====
+Rationale:: Long and complex Jinja templates can be difficult to maintain and debug. By splitting excessively long templates into logical componets that can be included as-needed, each template will be easier to maintain.
+====
+
+* Jinja templates should not be used to create structured data but instead text and semi-structured data.
+* Filter plugins are preferred over Jinja templates for the use of data manipulation or transformation.
```

Comment:
```suggestion
* Jinja templates should not be used to create structured data but instead text and semi-structured data.
  Filter plugins are preferred over Jinja templates for the use of data manipulation or transformation.
```

As I understand it now, this is actually one statement with the same rationale, so I would merge both bullet points into one.